index.html

```diff
@@ -163,10 +106,10 @@ <h2>Definition</h2>
 
       <p>This one <a>selection</a> must be shared by all the content of the <a>document</a>
         (though not by nested <a>document</a>s), including any editing hosts in the document.
-        <dfn>Editing hosts</dfn> (<a href="https://www.w3.org/TR/html5/editing.html#editing-host">defined</a> in [[!HTML5]]) that are not
+        <dfn data-cite="!HTML#editing-host">Editing hosts</dfn> that are not
```

Comment:
There are a bunch xrefs in this spec that would benefit from `<dfn data-cite>`. I only did a few, just to show how to do it.

Reply:
More info here:
https://github.com/w3c/respec/wiki/data--cite